modules/rosa-create-objects.adoc

```diff
@@ -741,6 +741,13 @@ $ rosa create machinepool --cluster=<cluster_name> | <cluster_id> --replicas=<nu
 a|--cluster <cluster_name>\|<cluster_id>
 |Required: The name or ID of the cluster to which the machine pool will be added.
 
+|--disk-size
```

Comment:
Out of scope, but would y'all consider making the items in the **Option** column `monospace` at some point since they are parameters for a command?